Formula/qbe.rb

```diff
@@ -0,0 +1,32 @@
+class Qbe < Formula
+  desc "Compiler Backend"
+  homepage "https://c9x.me/compile/"
+  url "https://c9x.me/compile/release/qbe-1.0.tar.xz"
+  sha256 "257ef3727c462795f8e599771f18272b772beb854aacab97e0fda70c13745e0c"
+  license "MIT"
+
+  def install
+    system "make", "PREFIX=#{prefix}"
+    system "make", "install", "PREFIX=#{prefix}"
+  end
+
+  test do
+    (testpath/"main.ssa").write <<~EOS
+      function w $add(w %a, w %b) {        # Define a function add
+      @start
+        %c =w add %a, %b                   # Adds the 2 arguments
+        ret %c                             # Return the result
+      }
+      export function w $main() {          # Main function
+      @start
+        %r =w call $add(w 1, w 1)          # Call add(1, 1)
+        call $printf(l $fmt, ..., w %r)    # Show the result
+        ret 0
+      }
+      data $fmt = { b "One and one make %d!\n", b 0 }
+    EOS
+
+    system "#{bin}/qbe", "-o", "out.s", "main.ssa"
+    assert_predicate testpath/"out.s", :exist?
```

Comment:
```suggestion
    assert_match ".globl _main", (testpath/"out.s").read
```

Reply:
@chenrui333 good call, though I think this might be flaky on different architectures. The CI seems to be failing in some cases since the generated assembly doesn't include a `_` prefix. I'm unsure why that is.

I reverted the commit for now, but it might be worth tackling this in the future.

Reply:
I see, that makes sense.